pkg/kubectl/resource_printer.go

```diff
@@ -259,7 +259,7 @@ func (h *HumanReadablePrinter) HandledResources() []string {
 var podColumns = []string{"NAME", "READY", "STATUS", "RESTARTS", "AGE"}
 var podTemplateColumns = []string{"TEMPLATE", "CONTAINER(S)", "IMAGE(S)", "PODLABELS"}
 var replicationControllerColumns = []string{"CONTROLLER", "CONTAINER(S)", "IMAGE(S)", "SELECTOR", "REPLICAS", "AGE"}
-var serviceColumns = []string{"NAME", "LABELS", "SELECTOR", "IP(S)", "PORT(S)", "AGE"}
+var serviceColumns = []string{"NAME", "CLUSTER IP", "EXTERNAL IP", "PORT(S)", "SELECTOR", "AGE"}
```

Comment:
In order for cut, etc. to work, we need a uniform number of columns, so please use CLUSTER_IP and EXTERNAL_IP.